pkgs/development/libraries/gettext/default.nix

```diff
@@ -49,6 +49,7 @@ stdenv.mkDerivation rec {
   buildInputs = stdenv.lib.optional (!stdenv.isLinux && !hostPlatform.isCygwin) libiconv;
 
   setupHook = ./gettext-setup-hook.sh;
+  gettextNeedsLdflags = !hostPlatform.libc != "glibc";
```

Comment:
double negation :)

Reply:
Thanks! Does this make sense to you though? It's an ugly variable name but it looks like the easiest way to check glibc in a setup-hook.